gpkit/nomials.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, exps=None, cs=1, varlocsandkeys=None,
                     units = cs[0]/cs[0].magnitude
                     if units.dimensionless:
                         cs = [c * ureg.dimensionless for c in cs]
+                        units = ureg.dimensionless
```

Comment:
Should we use pint's `to` method instead of growing our own?

```python
In [3]: ureg = pint.UnitRegistry()

In [4]: x = 3 * ureg.feet

In [5]: x
Out[5]: <Quantity(3, 'foot')>

In [6]: y = 1 * ureg.meter

In [7]: x/y
Out[7]: <Quantity(3.0, 'foot / meter')>

In [8]: (x/y).to(ureg.dimensionless)
Out[8]: <Quantity(0.9144, 'dimensionless')>
```

(I've barely played with pint; feel free to tell me this is a bad idea.)